onnxruntime/python/tools/transformers/models/llama/convert_to_onnx.py

```diff
@@ -0,0 +1,579 @@
+import argparse
+import logging
+import os
+import sys
+import tempfile
+from itertools import chain
+from typing import List
+
+import onnx
+import torch
+from transformers import LlamaConfig, LlamaForCausalLM
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", ".."))
+from benchmark_helper import Precision, prepare_environment, setup_logger  # noqa: E402
+from llama_inputs import get_sample_inputs, get_sample_with_past_kv_inputs  # noqa: E402
+from llama_parity import main as parity_check  # noqa: E402
+from onnx_model import OnnxModel  # noqa: E402
+
+from onnxruntime import quantization as ort_quantization  # noqa: E402
+
+logger = logging.getLogger("")
+
+
+def get_model_dynamic_axes(input_names: List[str], output_names: List[str]):
+    dynamic_axes = {}
+    for name in input_names + output_names:
+        if name in input_names:
+            # shape is (batch_size, sequence_length)
+            dynamic_axes[name] = {0: "batch_size", 1: "sequence_length"}
+        elif name == "logits":
+            # shape is (batch_size, sequence_length, vocab_size)
+            dynamic_axes[name] = {0: "batch_size", 1: "sequence_length"}
+        elif "present" in name:
+            # shape is (batch_size, num_heads, past_sequence_length + 1, head_size)
+            dynamic_axes[name] = {0: "batch_size", 2: "past_sequence_length + 1"}
```

Comment:
For model without past input, I think present shape is  (batch_size, num_heads, **sequence_length**, head_size)